benchbuild/project.py

```diff
@@ -30,7 +33,15 @@
 from benchbuild.extensions import compiler
 from benchbuild.extensions import run as ext_run
 from benchbuild.settings import CFG
-from benchbuild.utils import db, run, unionfs, wrapping
+from benchbuild.utils import db, run, unionfs, validators, wrapping
+
+if TYPE_CHECKING:
+    from benchbuild.experiment import Experiment
+    from benchbuild.extensions import Extension
+    from benchbuild.utils.container import Container
+    from plumbum.path.local import LocalPath
```

Comment:
C0412 (ungrouped-imports): Imports from package plumbum are not grouped  <!-- Automatically posted by Lintly -->